meson.build

```diff
@@ -0,0 +1,35 @@
+project('linthesia', 'cpp',
+  version : '0.7.0',
+  default_options : ['warning_level=3', 'cpp_std=c++14'])
+
+project_name = meson.project_name()
+
+common_flags = []
+if get_option('musicdir') == 'hello'
+  common_flags += [
+    '-DMUSICDIR=@0@'.format(get_option('musicdir'))
+  ]
+endif
+if get_option('graphdir') == 'hello'
+  common_flags += [
+    '-DGRAPHDIR=@0@'.format(get_option('graphdir'))
+  ]
+endif
+
+add_project_arguments(common_flags, language: 'cpp')
+
+i18n = import('i18n')
+gettext = project_name
+
+source_root = meson.current_source_dir()
+po_dir = join_paths(source_root, 'po')
+prefix = get_option('prefix')
+datadir = join_paths(prefix, get_option('datadir'))
+
+subdir('extra')
+install_subdir('music', install_dir: datadir / project_name )
```

Comment:
You can just do:

```meson
 install_subdir('music', install_dir: get_option('datadir') / project_name )
```
You don't need to prefix it with prefix, since install_dir allows relative paths and assumes they are relative to prefix already (and in fact, the introspection data works better if you use relative paths there).

Reply:
Didn't know